### PR TITLE
Adopt new netdata.conf defaults

### PR DESCRIPTION
**Pull request by filippocarletti**
https://github.com/NethServer/dev/issues/6429

### CODE REVIEW

File: root/etc/e-smith/templates/etc/netdata/netdata.conf/10base [hunk 1]
```diff
@@ -3,6 +3,10 @@
 
     # the default database size - 1 hour
     history = 3600
+    
+    # some defaults to run netdata with least priority
+    process scheduling policy = idle
+    OOM score = 1000
```
Comment (gsanchietti) [inline root/etc/e-smith/templates/etc/netdata/netdata.conf/10base:9]: Why these lines has been added?

**Review by gsanchietti — CHANGES_REQUESTED** [review]
The fix seems ok, but the patch contains unrelated changes.

Please also write a meaningful commit message.

File: root/etc/e-smith/templates/etc/netdata/netdata.conf/10base (hunk 1 above)
Comment (filippocarletti) [reply root/etc/e-smith/templates/etc/netdata/netdata.conf/10base:9]: See https://github.com/netdata/netdata/commit/55c10760e24d4d6cd7553e253237f7279aa4ea59

Comment (gsanchietti) [reply root/etc/e-smith/templates/etc/netdata/netdata.conf/10base:9]: I can't see nothing about OOM score on the linked commit. :thinking:

**Review by gsanchietti — APPROVED** [review]

**Pull request merged**